linux testAllExamples.sh fails

discovered a tiny bug in `testAllExamples.sh`
It stumbles over README.md (and eventual qtCreator build-* directories), and then fails.
```
./testAllExamples.sh: line 11: cd: README.md: Not a directory
-----------------------------------------------------------------
building  + README.md
./testAllExamples.sh: line 20: cd: README.md: Not a directory
make: *** No rule to make target 'Debug'.  Stop.
error compiling README.md
```


Since all Examples end with "Example"... would it be an idea to do something like this instead?

```
#!/bin/bash

export LC_ALL=C
SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
OF_ROOT=$SCRIPT_DIR/../..

THREADS=""
if [[ $1 == "-j*" ]]; then
  THREADS="$1"
fi

find $OF_ROOT/examples -type d \( -name android -o -name ios -o -name gles \) -prune -false -o -type d -name '*Example' -exec bash -c 'cd {} && make $THREADS && make run' \;

```
Also, why not make it possible to use multiple threads :)

I am never sure, is it a good idea to immediately create a PR for a small fix like this and link it, or is it better to first have it confirmed?

--- a/scripts/linux/testAllExamples.sh
+++ b/scripts/linux/testAllExamples.sh
@@ -11,7 +11,7 @@ do
     cd $category
     for example in $( ls . )
     do
-        if [[ "$example" == osx* ]]
+        if [[ "$example" == osx* ]] || [[ "$example" != *Example ]]
         then
             continue
         fi    

--- a/scripts/linux/testAllExamples.sh
+++ b/scripts/linux/testAllExamples.sh
@@ -5,7 +5,7 @@ cd ../../examples
 
 for category in $( ls . )
 do
-    if [ "$category" = "android" -o "$category" = "ios" -o "$category" = "gles" ]; then
+    if [ ! -d "$category" -o "$category" = "android" -o "$category" = "ios" -o "$category" = "gles" ]; then
             continue
     fi
     cd $category

